notification/notification-sdk/src/main/java/org/sunbird/notification/email/Email.java

```diff
@@ -246,6 +259,11 @@ private boolean sendEmail(Session session, MimeMessage message) {
       transport = session.getTransport("smtp");
       transport.connect(host, userName, password);
       transport.sendMessage(message, message.getAllRecipients());
+      logger.info("Host values is : " + host + "Username : " + userName + "pasword : " + password);
+      logger.info("Recepient list size : " + message.getAllRecipients().length + " Sender Value is : "  + message.getSender());
+      logger.info("Transport URL : " + transport.getURLName());
+      logger.info("Session Value SMTP Protocol : " + session.getProperty("mail.smtp.ssl.protocols"));
+      logger.info("Session value for TLS enabled: " + session.getProperty("mail.smtp.starttls.enable"));
```

Comment:
As mentioned in the other PR - remove the info logger

Reply:
This is already removed once previous PR is merged this should reflect